templates/ui-frameworks/svelte/web-app/ui/index.html.hbs

```diff
@@ -5,13 +5,11 @@
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Svelte hApp</title>
-
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
```

Comment:
There's still a material link left over here. And another one in `templates/custom-template/custom-template/template/webapp/ui/index.html.hbs`